Example/ConsentViewController_ExampleTests/SPClientCoordinator/SPClientCoordinatorSpec.swift

```diff
@@ -503,5 +503,33 @@ class SPClientCoordinatorSpec: QuickSpec {
                 }
             }
         }
+
+        // TODO: remove fdescribe
+        fdescribe("a property with USNat campaign") {
```

Comment:
FYI `f` in `fdescribe` might be a typo, pls double check

Reply:
Also, remove the comment pls

Reply:
I placed this comments on purpose, because unfortunately we have to skip a lot of tests in order to run them while pointing the SDK to stage. The SDK rely on ids that are only available in prod. Those `TODO`s will remind us to bring the tests back before merging to develop.